sofascoreAcademy/sofascoreAcademy/SettingsViewController.swift

```diff
@@ -9,27 +9,71 @@ import Foundation
 import UIKit
 import SofaAcademic
 import SnapKit
+import KeychainAccess
 
 class SettingsViewController: UIViewController, BaseViewProtocol {
     private let dismissButton = UIButton(type: .system)
+    private let nameLabel = UILabel()
+    private let logoutButton = UIButton()
+    private let eventCountLabel = UILabel()
+    private let leagueCountLabel = UILabel()
     
     override func viewDidLoad() {
         super.viewDidLoad()
         addViews()
         setupConstraints()
         styleViews()
+        countDB()
     }
     
     override func viewWillAppear(_ animated: Bool) {
         super.viewWillAppear(animated)
         navigationController?.setNavigationBarHidden(false, animated: true)
     }
     
+    private func countDB() {
+        let eventCount = try? DbManager.shared.dbQueue?.read { db in
+            try? DBEvent.fetchAll(db).count
+        }
+        
+        let leagueCount = try? DbManager.shared.dbQueue?.read { db in
+            try? DBLeague.fetchAll(db).count
+        }
```

Comment:
Nebi se smjelo ovako baza pozivati direkt u VC...napravi StorageManager koji će wrapati DbManager i vratiti ono što ti treba sa async/await funkcijama... znači da ti sintaksa bude isto kao da zoveš ApiClient.

Onda ćeš zapravo sve operacije s bazom imati na jednom mjestu,..